### DISCUSSION TITLE
Reports - Others output format support

### DISCUSSION THREAD

**Discussion by mauroalexandre** [frangoteam/FUXA#538] [Ideas]
Hi!

When you use data from FUXA to integrate with another platforms other output formats in reports very welcome.

In particular maybe we can start support for CSV and JSON.

**Comment by unocelli** [comment]
Hi,
what do you mean by output formats?

**Comment by mauroalexandre** [comment]
Hi @unocelli !

File output formats. Today the report is generated just in PDF, right?

**Reply by unocelli** [reply]
Yes, and the report are created with the pdfmake library to which you pass the content in json format, but the question is more how the format should be structured, I guess it should then be imported from other applications.
